### ISSUE TITLE
local flash doesn't work on 16.6.4 (Invalid device: "/dev/sdb")

### ISSUE THREAD

**Issue by acegla** [balena-io/balena-cli#2644]
Hi, on latest release **local flash** doesn't work. On earlier version (I've tested on v14.5.18) works without issue on the same machine, same card.

---

# Expected Behavior

flash img to sd card

# Actual Behavior

```
sudo balena-cli/balena local flash Pulpit/2023-05-03-raspios-bullseye-armhf-lite.img
? Select a drive /dev/sdb (31.9 GB) - Generic Mass-Storage (bootfs)
? This will erase the selected drive. Are you sure? Yes
Invalid device: "/dev/sdb"
```

# Steps to Reproduce the Problem

Download *balena-cli-v16.6.4-linux-x64-standalone.zip*
run **balena local flash**

# Specifications

- **balena CLI version:** 
```
balena-cli version "16.6.4"
Node.js version "16.16.0"
```
- **Operating system version:** 
```
cat /etc/os-release
NAME="Linux Mint"
VERSION="20.3 (Una)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 20.3"
VERSION_ID="20.3"
```
- **32/64 bit OS and processor:** e.g. 64-bit linux on 64-bit Intel processor
- **Install method:* zip package

# Additional References
### working on 14.5.18
```
$ balena_14.5.18/balena-cli/balena local flash Pulpit/2023-05-03-raspios-bullseye-armhf-lite.img
? Select a drive /dev/sdb (31.9 GB) - Generic Mass-Storage (bootfs)
? This will erase the selected drive. Are you sure? Yes
Flashing [========================] 100% eta 0s
Validating [========================] 100% eta 0s
```
### doesn't work on 16.6.4
```
$ sudo balena-cli/balena local flash Pulpit/2023-05-03-raspios-bullseye-armhf-lite.img --debug
[debug] new argv=[/home/asus/balena-cli/balena,/snapshot/balena-cli/bin/balena,local:flash,Pulpit/2023-05-03-raspios-bullseye-armhf-lite.img] length=4
[debug] Deprecation check: 0.07792 days since last npm registry query for next major version release date.
[debug] Will not query the registry again until at least 7 days have passed.
[debug] Event tracking error: Timeout awaiting 'response' for 0ms
? Select a drive /dev/sdb (31.9 GB) - Generic Mass-Storage (bootfs, rootfs)
? This will erase the selected drive. Are you sure? Yes
Invalid device: "/dev/sdb"
```

tested with and without sudo

**Comment by tampe125** [comment]
Hello,  
I'd wish to report that I'm having the same problem with 16.7.6 .  
Tried with and without sudo but I still have the same issue

**Comment by otaviojacobi** [comment]
Hello  @acegla and @tampe125 Thanks for the report. A fix should be merged soon with #2669

**Comment by otaviojacobi** [comment]
Fixed by #2669 local flash should be working again on balena-cli >= 17.1.1

**Comment by tampe125** [comment]
I can confirm the fix. Thank you!
